README.md

```diff
@@ -1,13 +1,99 @@
-# Substrate API Sidecar
 
-REST API service intended to run next to Substrate, exposing a limited set of endpoints over HTTP
-with meaningful responses.
+<br /><br />
 
-## Installation
+<div align="center">
+  <h1 align="center">@substrate/api-sidecar</h1>
+  <h4 align="center"> REST service that makes it easy to interact with blockchain nodes built using Substrate's FRAME framework.</h4>
 
-Make sure your machine has an
-[up-to-date version of `rustup`](https://www.rust-lang.org/tools/install) installed to manage Rust
-dependencies.
+  <p align="center">
+    <a href="https://www.npmjs.com/package/@substrate/api-sidecar">
+      <img alt="npm" src="https://img.shields.io/npm/v/@substrate/api-sidecar" />
+    </a>
+    <a href="https://github.com/paritytech/substrate-api-sidecar/actions">
+      <img alt="Github Actions" src="https://github.com/paritytech/substrate-api-sidecar/workflows/pr/badge.svg" />
+    </a>
+    <a href="https://github.com/paritytech/substrate-api-sidecar/blob/master/LICENSE">
+      <img alt="GPL-3.0-or-later" src="https://img.shields.io/npm/l/@substrate/api-sidecar" />
+    </a>
+  </p>
+</div>
+
+<br /><br />
+
+## Note
+
+This a release candidate for v1.0.0 and it contains both v0 and v1 endpoints. **`substrate-api-sidecar`
+releases after Friday, October 16th, 2020, will not make any guarantees to include deprecated v0
+endpoints.** It is important that users complete the transition to the new endpoints by this date so
+they are ready for any subsequent emergency updates. Please visit the [MIGRATION_GUIDE](/MIGRATION_GUIDE.md) to
+learn more.
+
+## Perquisites
+
+This service requires Node version 12 or higher.
+
+## Table of contents
+
+- [NPM package installation and usage](#npm-package-installation-and-usage)
+- [Source code installation and usage](#source-code-installation-and-usage)
+- [Running](#running)
+- [Configuration](#configuration)
+- [Debugging fee and payout calculations](#debugging-fee-and-payout-calculations)
+- [Available endpoints](https://paritytech.github.io/substrate-api-sidecar/dist/)
```

Comment:
We could link to within the README for continuity, but that section just has the same link